src/icon.rs

```diff
@@ -123,7 +139,36 @@ impl Icon {
     /// `rgba.len() / 4`. Otherwise, this will return a `BadIcon` error.
     pub fn from_rgba(rgba: Vec<u8>, width: u32, height: u32) -> Result<Self, BadIcon> {
         Ok(Icon {
-            inner: PlatformIcon::from_rgba(rgba, width, height)?,
+            inner: PlatformIcon::from_rgba(rgba, width, height, 0, 0)?,
+        })
+    }
+}
+
+#[derive(Clone)]
+pub struct CustomCursorIcon {
+    pub(crate) inner: PlatformCustomCursorIcon,
+}
+
+impl fmt::Debug for CustomCursorIcon {
+    fn fmt(&self, formatter: &mut fmt::Formatter<'_>) -> Result<(), fmt::Error> {
+        fmt::Debug::fmt(&self.inner, formatter)
+    }
+}
+
+impl CustomCursorIcon {
+    pub fn from_rgba(rgba: Vec<u8>, width: u32, height: u32) -> Result<Self, BadIcon> {
+        Self::from_rgba_with_hotspot(rgba, width, height, width / 2, height / 2)
```

Comment:
```suggestion
        Self::from_rgba_with_hotspot(rgba, width, height, 0, 0)
```
Shouldn't the default be the top-left corner?